vpc/tests/unit.tftest.hcl

```diff
@@ -56,6 +56,10 @@ run "aws_vpc_unit_test" {
     error_message = "Should be: String"
   }
 
+  assert {
+    condition     = aws_ssm_parameter.combined_nat_gateway_eips.tags.Copilot-application == "__all__"
```

Comment:
```suggestion
    condition     = aws_ssm_parameter.combined_nat_gateway_eips.tags.Copilot-application == "__all__"
    error message = "Should be: __all__"
```

Reply:
Hard to get formatting right in github suggestion, but that's why codebuild terraform-tests is failing. And I think it needs tf lint or whatever running again